### PR TITLE
Precompile templates into render functions

### PR DESCRIPTION
**Pull request by 4ver**

### CODE REVIEW

**Comment by joerick** [comment]
Magic! I was just looking for this. I'm gonna use your fork for now @4ver

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "author": "visualfanatic",
   "main": "index.js",
   "dependencies": {
-    "svgo": "^0.7.1"
+    "svgo": "^0.7.1",
+    "vue-template-compiler": "^2.4.2"
```
Comment (joerick) [inline package.json]: Should this be a peer dependency or something? It didn't match the version of Vue I had installed on my project.

Comment (4ver) [inline package.json]: Oh, possibly yeah. What version of vue/vue-loader are you using?

Is published here for now till this gets merged: https://www.npmjs.com/package/@teamwork/vue-svg-loader

Comment (4ver) [inline package.json]: Just published 1.0.2. Has vue-template-compiler moved to peer dependencies.

**Comment by visualfanatic** [comment]
Thank you!

**Comment by danciupuliga** [comment]
this is great! I was looking for it. Any idea when it will be published to NPM?

**Comment by visualfanatic** [comment]
@danciupuliga should be in a few hours

**Pull request merged**